src/components/TooltipIconButton/TooltipIconButton.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ThemeProvider } from '@material-ui/core';
+import theme from 'theme';
+
+import IconButton from '@material-ui/core/IconButton';
+import SearchIcon from '@material-ui/icons/SearchOutlined';
+import Component from './TooltipIconButton';
+
+export default {
+    title: 'Components/TooltipIconButton',
+    component: Component,
+};
+
+export function TooltipIconButton() {
+    return (
+        <ThemeProvider theme={theme}>
```

Comment:
Same thing here about the ThemeProvider.

Reply:
Same as TextField